.rubocop_todo.yml

```diff
@@ -256,6 +212,13 @@ Naming/VariableNumber:
 Rails/AssertNot:
   Enabled: false
 
+# Offense count: 96
+# Configuration parameters: Database, Include.
+# SupportedDatabases: mysql, postgresql
+# Include: db/migrate/*.rb
+Rails/BulkChangeTable:
```

Comment:
Where is this coming from now? :thinking:

Reply:
yeah, I'm using PG :-)